src/main/java/com/wiley/autotest/spring/SeleniumTestExecutionListener.java

```diff
@@ -102,124 +58,138 @@ public void prepareTestInstance(final TestContext context) throws Exception {
         count.set(count.get() + 1);
         driverRestartCount.set(driverRestartCount.get() + 1);
 
+        //TODO NT - confirm if it the right place to call this method?
+        currentAlertCapability.set(alertCapability.get());
```

Comment:
I think need move this to beforeTestMethod after 'if' where check alertCapability

Reply:
@estuold we probably need to discuss how can we make changing different capabilities on-flight. not only the alert but something more general. (and then to use alert as a particular case)